RecoTracker/MeasurementDet/plugins/MeasurementTrackerESProducer.cc

```diff
@@ -37,6 +38,16 @@ using namespace edm;
 MeasurementTrackerESProducer::MeasurementTrackerESProducer(const edm::ParameterSet & p) 
 {  
   std::string myname = p.getParameter<std::string>("ComponentName");
+  pixelCPEName = p.getParameter<std::string>("PixelCPE");
+  stripCPEName = p.getParameter<std::string>("StripCPE");
+  matcherName  = p.getParameter<std::string>("HitMatcher");
+
+  //FIXME:: just temporary solution for phase2!
+  phase2TrackerCPEName = "";
+  if (p.existsAs<std::string>("Phase2StripCPE")) {
+    phase2TrackerCPEName = p.getParameter<std::string>("Phase2StripCPE");
+  }
+
   pset_ = p;
```

Comment:
ehm, I should have looked at more than just context lines of the original commit.
There are many more issues of the same kind in the ::produce.
So, it's nice that the number of pset.get\* calls in ::produce reduced, but the issue remains.